charts/k8s-service/README.md

```diff
@@ -965,6 +966,44 @@ Note that certain changes will lead to a replacement of the `Deployment` resourc
 `applicationName` will cause the `Deployment` resource to be deleted, and then created. This can lead to down time
 because the resources are replaced in an uncontrolled fashion.
 
+## How do I create a canary deployment? 
+
+You may optionally configure a [canary deployment](https://martinfowler.com/bliki/CanaryRelease.html) of an arbitrary tag that will run as an individual deployment behind your configured service. This is useful for ensuring a new application tag runs without issues prior to fully rolling it out. 
+
+To configure a canary deployment, set `canary.enabled = true` and define the `containerImage` values. Typically, you will want to specify the tag of your next release candidate:
+
+```yaml
+canary:
+  enabled: true
+    containerImage:
+      repository: nginx
+      tag: 1.15.9 
+```
+Once deployed, your service will route traffic across both your stable and canary deployments, allowing you to monitor for and catch any issues early.
+
+## How do I verify my canary deployment? 
+
+Canary deployment pods have the same name as your stable deployment pods, with the addition "-canary" appended to the end, like so: 
```

Comment:
```suggestion
Canary deployment pods have the same name as your stable deployment pods, with the addition `-canary` appended to the end, like so: 
```

Reply:
Super seded by Josh's suggestion down below.

Reply:
Fixed in 5957d07